sdk/webpubsub/azure-messaging-webpubsub-client/src/main/java/com/azure/messaging/webpubsub/client/WebPubSubClient.java

```diff
@@ -0,0 +1,329 @@
+     * @return the result.
+     */
+    public WebPubSubResult sendToGroup(String group, String content) {
+        return sendToGroup(group, BinaryData.fromString(content), WebPubSubDataType.TEXT);
+    }
+
+    /**
+     * Sends message to group.
+     *
+     * @param group the group name.
+     * @param content the data as WebPubSubDataType.TEXT.
+     * @param options the options.
+     * @exception SendMessageFailedException thrown if client not connected, or send group message failed.
+     * @return the result.
+     */
+    public WebPubSubResult sendToGroup(String group, String content, SendToGroupOptions options) {
+        return sendToGroup(group, BinaryData.fromString(content), WebPubSubDataType.TEXT, options);
+    }
+
+    /**
+     * Sends message to group.
+     *
+     * @param group the group name.
+     * @param content the data.
+     * @param dataType the data type.
+     * @exception SendMessageFailedException thrown if client not connected, or send group message failed.
+     * @return the result.
+     */
+    public WebPubSubResult sendToGroup(String group, BinaryData content, WebPubSubDataType dataType) {
+        return client.sendToGroup(group, content, dataType).block();
+    }
+
+    /**
+     * Sends message to group.
+     *
+     * @param group the group name.
+     * @param content the data.
+     * @param dataType the data type.
+     * @param options the options.
+     * @exception SendMessageFailedException thrown if client not connected, or send group message failed.
+     * @return the result.
+     */
+    public WebPubSubResult sendToGroup(String group, BinaryData content, WebPubSubDataType dataType,
+                                       SendToGroupOptions options) {
+        return client.sendToGroup(group, content, dataType, options).block();
+    }
+
+    /**
+     * Sends event.
+     *
+     * @param eventName the event name.
+     * @param content the data.
+     * @param dataType the data type.
+     * @exception SendMessageFailedException thrown if client not connected, or send group message failed.
+     * @return the result.
+     */
+    public WebPubSubResult sendEvent(String eventName, BinaryData content, WebPubSubDataType dataType) {
+        return client.sendEvent(eventName, content, dataType).block();
+    }
+
+    /**
+     * Sends event.
+     *
+     * @param eventName the event name.
+     * @param content the data.
+     * @param dataType the data type.
+     * @param options the options.
+     * @exception SendMessageFailedException thrown if client not connected, or send group message failed.
+     * @return the result.
+     */
+    public WebPubSubResult sendEvent(String eventName, BinaryData content, WebPubSubDataType dataType,
+                                           SendEventOptions options) {
+        return client.sendEvent(eventName, content, dataType, options).block();
+    }
+
+    WebPubSubClientState getClientState() {
```

Comment:
I guess it's for test purpose? Do we want to expose them in our public client?

Reply:
I assume package private would be OK. (it kind of hidden from user)

Any better idea?

Reply:
I guess some comments on the methods could help. 

If it's for test only, I think we could just use reflection to get the inner client. (Just my own opinion..)

Reply:
I tried to ask Chatgpt on this :)
![image](https://user-images.githubusercontent.com/87355844/229042953-199c5404-b82f-4ff1-9ff5-d20150f97671.png)

Reply:
Got it. Added comment.

The ctor is also private package. Hence I will leave it here. Reflection seems overkill.